routers/api/v1/repo/release_attachment.go

```diff
@@ -182,7 +181,7 @@ func CreateReleaseAttachment(ctx *context.APIContext) {
 	}
 
 	// Check if the filetype is allowed by the settings
-	err = upload.VerifyAllowedContentType(buf, strings.Split(setting.AttachmentAllowedTypes, ","))
+	err = upload.Verify(buf, header.Filename, setting.AttachmentAllowedTypes)
```

Comment:
Shouldn't this pass empty list to Verify so that it allows everything to be uploaded as release artifact?

Reply:
Actually, I think we either need to remove the verification for releases or add a new option for it. Currently it's shared with issue/pr attachments.